src/servers/src/http.rs

```diff
@@ -336,6 +336,20 @@ impl Display for Epoch {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum RequestSource {
+    Dashboard,
+}
+
+impl RequestSource {
+    pub fn parse(s: &str) -> Option<Self> {
+        match s {
+            "dashboard" => Some(RequestSource::Dashboard),
+            _ => None,
+        }
+    }
+}
```

Comment:
Consider using strum `EnumString` and the `#[strum(ascii_case_insensitive)]` attribute. Then we could get rid of the `to_lower_case()` call.
https://docs.rs/strum/latest/strum/derive.EnumString.html